backend-services/src/data_service/operations/dataschema.py

```diff
@@ -12,7 +12,9 @@ async def create_dataschema(engine: Engine, params: CreateDataschema) -> Datasch
     Create a new dataschema.
     """
     new_dataschema = Dataschema(
-        wallet_id=params.wallet_id, data_pool_id=params.data_pool_id, name=params.name, description=params.description, length=params.length, created=params.created
+        name=params.name,
+        data_schema=params.data_schema,
```

Comment:
I think we should also store "schema_description" or just "description" and send it with any schema request. Will be needed to display schemas to users.